lib/commands.coffee

```diff
@@ -6,6 +6,7 @@ class Commands
     @_subscriptions.add atom.commands.add 'atom-workspace',
       'linter:next-error': @nextError.bind(@)
       'linter:toggle': @toggleLinter.bind(@)
+      'linter:set-bubble-transparent': @setBubleTransparent
```

Comment:
Well, he didn't mean that.
he meant
```coffee
'linter:set-bubble-transparent': => @setBubleTransparent
```
and can you also revert the change to fat arrow below